From 8906954a17adbee2e0cce880b5cb73e1fe459a67 Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 2 Feb 2016 22:45:35 +0800
Subject: [PATCH 38/50] nx503a: ARM:dts:msm: update generic cam entries

cherry-pick from 7eaf475ecdf649badc5cea5aa6c601ede5130a80

remove kernel sensors drivers for imx135.

Signed-off-by: BambooIV <ningdisheng@gmail.com>
---
 drivers/media/platform/msm/camera_v2/sensor/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/media/platform/msm/camera_v2/sensor/Makefile b/drivers/media/platform/msm/camera_v2/sensor/Makefile
index 6365592..3c4e1d7 100755
--- a/drivers/media/platform/msm/camera_v2/sensor/Makefile
+++ b/drivers/media/platform/msm/camera_v2/sensor/Makefile
@@ -5,8 +5,6 @@ ccflags-y += -Idrivers/media/platform/msm/camera_v2/sensor/io
 ccflags-y += -Idrivers/media/platform/msm/camera_v2/sensor/cci
 obj-$(CONFIG_MSMB_CAMERA) += cci/ io/ csiphy/ csid/ actuator/ flash/ eeprom/
 obj-$(CONFIG_MSM_CAMERA_SENSOR) += msm_sensor_init.o msm_sensor_driver.o msm_sensor.o
-obj-$(CONFIG_IMX135_Z5S) += imx135_z5s.o 
-obj-$(CONFIG_IMX135_Z5S_069) += imx135_z5s_069.o 
 obj-$(CONFIG_IMX134) += imx134.o
 obj-$(CONFIG_OV8825) += ov8825.o
 obj-$(CONFIG_OV8865) += ov8865.o
-- 
1.9.1

